pkg/middleware/validation.go

```diff
@@ -45,31 +47,60 @@ func NewValidationMiddleware(validatorHook string) mux.MiddlewareFunc {
 	}
 }
 
+func maybeLogValidationReason(valid ValidationResponse, r *http.Request, mod string, version string) {
+	message := valid.ValidationMessage
+	if message.Reason != "" {
+		entry := log.EntryFromContext(r.Context())
+		entry.Warnf("Module %s:%s Reason: %s, Description: %s", mod, version, message.Reason, message.Description)
```

Comment:
It's a bit of an unwritten convention to do `mod@ver` instead of `mod:ver`

Reply:
Changed the format: https://github.com/gomods/athens/pull/1517/commits/248709afc8f032bee8a6cdd4818ffc02663566fd